src/modules/tests/test-data/cat-home-test/vehicle-checks/vehicle-checks.cat-home-test.action.ts

```diff
@@ -0,0 +1,38 @@
+import { Action } from '@ngrx/store';
+import { QuestionResult, QuestionOutcome } from '@dvsa/mes-test-schema/categories/common';
+
+export const SHOW_ME_QUESTION_SELECTED = '[VehicleChecksPage] [CatHomeTest] Show Me Question Selected';
+export const SHOW_ME_QUESTION_OUTCOME_CHANGED = '[VehicleChecksPage] [CatHomeTest] Show Me Question Outcome Changed';
+export const TELL_ME_QUESTION_SELECTED = '[VehicleChecksPage] [CatHomeTest] Tell Me Question Selected';
+export const TELL_ME_QUESTION_OUTCOME_CHANGED = '[VehicleChecksPage] [CatHomeTest] Tell Me Question Outcome Changed';
+export const ADD_SHOW_ME_TELL_ME_COMMENT = '[Vehicle Checks] [CatHomeTest] Add Show me Tell me comment';
+
```

Comment:
Missing InitializeVehicleChecks - this is mandatory for starting tests & will be needed to unblock the rest of the sprint, please don't forget this

Reply:
no longer applicable